example/tests/accessibility.spec.ts

```diff
@@ -42,26 +43,57 @@ describe('Accessibility tests', () => {
     const state = await appConnection.getAccessibilityState('slider');
     expect(state.accessible).toBeTruthy();
     expect(state.handler).toBeTruthy();
-    expect(parseFloat (state.display)).toEqual(0);
+    expect(parseFloat(state.display)).toEqual(0);
   });
 
   it('Slider display changes with the value change', async () => {
-    expect(parseFloat ((await appConnection.getAccessibilityState('slider')).display)).toEqual(0);
+    expect(
+      parseFloat((await appConnection.getAccessibilityState('slider')).display)
+    ).toEqual(0);
 
     await appConnection.clickComponent('increment-button');
-    expect(parseFloat ((await appConnection.getAccessibilityState('slider')).display)).toEqual(1);
+    expect(
+      parseFloat((await appConnection.getAccessibilityState('slider')).display)
+    ).toEqual(1);
 
     await appConnection.clickComponent('decrement-button');
-    expect(parseFloat ((await appConnection.getAccessibilityState('slider')).display)).toEqual(0);
+    expect(
+      parseFloat((await appConnection.getAccessibilityState('slider')).display)
+    ).toEqual(0);
   });
 
   it('Increment button has accessibility parent', async () => {
-    expect ((await appConnection.getAccessibilityParent('increment-button'))).toEqual('main-window');
-    expect ((await appConnection.getAccessibilityParent('enable-button'))).toEqual('main-window');
-    expect ((await appConnection.getAccessibilityParent('slider'))).toEqual('main-window');
+    expect(
+      await appConnection.getAccessibilityParent('increment-button')
+    ).toEqual('main-window');
+    expect(await appConnection.getAccessibilityParent('enable-button')).toEqual(
+      'main-window'
+    );
+    expect(await appConnection.getAccessibilityParent('slider')).toEqual(
+      'main-window'
+    );
   });
 
   it('Decrement button has no accessibility parent', async () => {
-    expect ((await appConnection.getAccessibilityParent('decrement-button'))).toEqual('');
+    expect(
+      await appConnection.getAccessibilityParent('decrement-button')
+    ).toEqual('');
+  });
+
+  it('Main window has multiple accessible children', async () => {
+    const children = await appConnection.getAccessibilityChildren(
+      'main-window'
+    );
+    expect(children.length).toEqual(5);
```

Comment:
Maybe just check that it's greater than 0? It will be a pain to update this if someone adds another child